app/views/govuk_publishing_components/components/_big_number.html.erb

```diff
@@ -0,0 +1,19 @@
+<%
+  items ||= []
+%>
+<% if items.any? %>
+  <div class="gem-c-big-number">
+    <% if items.length > 1 %>
```

Comment:
Also the component sets the spacing between more than one number. What if that doesn't match the required design? Then we need an option for the layout (horizontal, vertical) and options for the space between them. Only rendering one number seems simpler.

Reply:
So my thoughts on this are:

- On govuk presently, we use this component as a stacked list more often than we don't. [Example 1](https://www.gov.uk/government/how-government-works). [Example 2](https://www.gov.uk/government/get-involved). [Example 3](https://www.gov.uk/). I feel that it creates risk by not enforcing a list structure for multiple numbers, creating an (albeit quite minor) accessibility risk and effectively outsourcing the messiness across all our apps. This, whilst involving some extra calculation in the component, keeps the apps simple which is where the risk of things becoming messier is higher, in my opinion.
- I feel that the spacing is adequate both for between numbers and for between it and proceeding elements. I reckon that your suggestion for a custom margin is probably a good idea, which also is a case for only having this be one number that devs call as and when, but also I'm keen not to build in features that aren't necessary _right now_.

I'm on the fence about making this a single component in all honestly. Interested to know your thoughts on the above.

Reply:
I didn't realise we had that many instances of multiple numbers, but I still think it should be a single thing. I think we can trust developers to do the right thing (and the list styles in question are already included in every application).

Absolutely, we shouldn't build in features that aren't needed right now, I'm thinking ahead to the complexity of unpicking this if e.g. different spacing is required, or an ordered list is required instead of an unordered one. Having it as a single thing makes it much more flexible, and reduces its complexity significantly.